packages/generator-one-app-module/generators/app/templates/base-module/__tests__/components/ModuleContainer.spec.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import toJson from 'enzyme-to-json';
+import AppConfig from '../../src/appConfig';
+import <%=moduleNamePascal%> from '../../src/components/<%=moduleNamePascal%>';
+
+configure({ adapter: new Adapter() });
```

Comment:
shouldn't this be done in a setup script?

Reply:
fixed in 6ca9aa7. Added `enzyme.config.js` within the `__tests__` folder, and told jest to ignore trying to run tests against the file (since it's just config) in `package.json`:
```
    "testPathIgnorePatterns": [
      "./__tests__/enzyme.config.js"
    ]
```
This block can be removed if we put the `enzyme.config.js` at the root level, but to me this was disorganized file structure. Let me know if you agree, or want to move it to root.

Reply:
I would move to the root

Reply:
moved to root in 4f6eb94